seed/seed.json

```diff
@@ -913,7 +913,7 @@
                 }
             ],
             "filter": {
-                "channel": ["BETA", "NIGHTLY"],
```

Comment:
typically we should specify min_version, to not enable it for old clients @bsclifton @rebron

Reply:
also @kjozwiak

Reply:
@iefremov can we get this fixed or is it too late at this point? What would happen if a client that ended up pulling the above study doesn't meet the new requirements. Would it simply be removed? Assuming it wouldn't cause any issues for users that meet the requirements and have the study?

I'll make sure we have a `min_version` next time 👍

Reply:
Thanks for the heads up - will do that going forward. No action needed for this, AFAIK. But they would be shown as being in an experiment which isn't applicable